src/ClientBuilder.php

```diff
@@ -41,18 +41,12 @@
  * @method setSampleRate(float $sampleRate)
  * @method string getMbDetectOrder()
  * @method setMbDetectOrder(string $detectOrder)
- * @method bool getAutoLogStacks()
- * @method setAutoLogStacks(bool $enable)
+ * @method bool getAttachStacktrace()
+ * @method setAttachStacktrace(bool $enable)
  * @method int getContextLines()
  * @method setContextLines(int $contextLines)
- * @method string getCurrentEnvironment()
- * @method setCurrentEnvironment(string $environment)
- * @method string[] getEnvironments()
- * @method setEnvironments(string[] $environments)
- * @method string[] getExcludedLoggers()
- * @method setExcludedLoggers(string[] $loggers)
- * @method string[] getExcludedExceptions()
```

Comment:
You didn't remove `get`/`setExcludedExceptions`

Reply:
fixed